src/main/java/com/revature/studyforce/notification/controller/SubscriptionController.java

```diff
@@ -0,0 +1,155 @@
+    }
+
+    /**
+     * DELETE mapping for '/deleteFlashcardSubscription in {@link FlashcardSubscriptionService#deleteFlashcardSubscription(Integer, Integer)}
+     * @param subscriptionRequest The DTO with the flashcardId and the userId
+     * @return The users flashcard subscription that was deleted
+     */
+    @DeleteMapping("/flashcards")
+    public @ResponseBody
+    FlashcardSubscription deleteFlashcardSubscription(@RequestBody FlashcardSubscriptionDTO subscriptionRequest){
+        return flashcardSubscriptionService.deleteFlashcardSubscription(
+                subscriptionRequest.getFlashcardId(),
+                subscriptionRequest.getUserId()
+        );
+    }
+
+    /**
+     * Processes POST request on '/subscriptions/stacktraces' to create a stacktrace subscription {@link StacktraceSubscriptionService#createStacktraceSubscription(Integer, Integer)}
+     * @param subscriptionRequest A data transfer object containing the stacktrace subscription information
+     * @return The created subscription
+     */
+    @PostMapping("/stacktraces")
+    public @ResponseBody
+    StacktraceSubscription createStacktraceSubscription(@RequestBody StacktraceSubscriptionDTO subscriptionRequest){
+        return stacktraceSubscriptionService.createStacktraceSubscription(
+                subscriptionRequest.getStacktraceId(),
+                subscriptionRequest.getUserId());
+    }
+
+    /**
+     * GET mapping for '/getStacktraceSubscriptionsByStacktraceAndUserId in {@link StacktraceSubscriptionService#getStacktraceSubscriptionByStacktraceAndUserId(Integer, Integer)}
+     * @param stacktraceId The stacktrace id of the stacktrace to find the stacktrace
+     * @param userId The user id of the user to find the subscription for the user
+     * @return The users stacktrace subscription
+     */
+    @GetMapping("/stacktraces")
+    public @ResponseBody
+    StacktraceSubscription getStacktraceSubscription(@RequestParam("stacktrace-id") String stacktraceId,
+                                                     @RequestParam("user-id") String userId){
+        return stacktraceSubscriptionService.getStacktraceSubscriptionByStacktraceAndUserId(
+                Integer.parseInt(stacktraceId),
+                Integer.parseInt(userId)
+        );
+    }
+
+    /**
+     * DELETE mapping for '/deleteStacktraceSubscription in {@link StacktraceSubscriptionService#deleteStacktraceSubscription(Integer, Integer)}
+     * @param subscriptionRequest The DTO with the stacktraceId and the userId
+     * @return The users stacktrace subscription that was deleted
+     */
+    @DeleteMapping("/stacktraces")
+    public @ResponseBody
+    StacktraceSubscription deleteStacktraceSubscription(@RequestBody StacktraceSubscriptionDTO subscriptionRequest){
+        return stacktraceSubscriptionService.deleteStacktraceSubscription(
+                subscriptionRequest.getStacktraceId(),
+                subscriptionRequest.getUserId()
+        );
+    }
+
+}
```

Comment:
fix linting issues
mapping needs tweaked to meet rest standards

Reply:
Looking into it